packages/backend/scripts/get-op-schema.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+## https://stackoverflow.com/questions/59895/how-do-i-get-the-directory-where-a-bash-script-is-located-from-within-the-script
+SOURCE=${BASH_SOURCE[0]}
+while [ -L "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink
+  DIR=$( cd -P "$( dirname "$SOURCE" )" >/dev/null 2>&1 && pwd )
+  SOURCE=$(readlink "$SOURCE")
+  [[ $SOURCE != /* ]] && SOURCE=$DIR/$SOURCE # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
+done
+OUTDIR=$( cd -P "$( dirname "$SOURCE" )/../src/openapi" >/dev/null 2>&1 && pwd )
+
+curl -o "$OUTDIR/resource-server.yaml" https://raw.githubusercontent.com/interledger/open-payments/b363d33038fe789e5388f04f80ddd06a4fa97093/openapi/resource-server.yaml
+curl -o "$OUTDIR/schemas.yaml" https://raw.githubusercontent.com/interledger/open-payments/b363d33038fe789e5388f04f80ddd06a4fa97093/openapi/schemas.yaml
```

Comment:
Can we also update the `open-payments` commit version in this PR

_or_ would it make sense to export these from `open-payments`? so we can be sure both are using the same spec? This does seem vaguely familiar though... have we discussed this before?

Reply:
We could even do it for both of `backend` and `auth` projects since `open-payments` contains both, which would eliminate the need for the `packages/backend/src/openapi/auth-server.yaml` dependency

Reply:
>would it make sense to export these from open-payments?

I'm favor of having defining the (default) commit.
It seems like that could happen while still allowing subsequent local tinkering of the specs
https://github.com/interledger/rafiki/pull/684#discussion_r1009661953

>which would eliminate the need for the packages/backend/src/openapi/auth-server.yaml dependency

Did you mean `resource-server.yaml`?